phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/breezeDark/main.css

```diff
@@ -218,11 +218,15 @@
 /*******************************************************/
 /************************ logo *************************/
 /*******************************************************/
-#logoText {
+#logo {
+    -fx-background-image: url('/org/phoenicis/javafx/views/common/phoenicis.png');
+    -fx-background-size: 1.8em;
+    -fx-background-repeat: no-repeat;
+    -fx-background-position: left center;
+    -fx-label-padding: 0 0 0 2.2em;
+    -fx-vpos: center;
     -fx-font-family: "Maven Pro";
     -fx-font-size: 1.8em;
```

Comment:
Ok, then I'm out of ideas.
Maybe it's because the image is shown inside a tab header?
It would be nice to see if the same java code and css would work, if the icon is added to another element via `setContent` or `getChildren().add`.